### ISSUE TITLE
a question about the repo

### ISSUE THREAD

**Issue by outranker** [gboutry/.config-nvim#1]
Hi!
I was just going through nvim configurations on github and came across this repo. 
I was wondering if Prime did some series/tutorial somewhere that showed what you wrote on this repo's description ie from 0 to rc.

I myself currently playing with vim and see if it suits me

**Comment by gboutry** [comment]
Yes, here's the [link](https://www.youtube.com/watch?v=w7i4amO_zaE)
